js/bitfinex.js

```diff
@@ -379,10 +379,13 @@ module.exports = class bitfinex extends Exchange {
                     'spot': 'exchange',
                     'margin': 'trading',
                     'funding': 'deposit',
-                    'exchange': 'exchange',
-                    'trading': 'trading',
-                    'deposit': 'deposit',
-                    'derivatives': 'trading',
+                    'future': 'exchange',
+                    'contract': 'trading',
```

Comment:
Here I wasn't sure what unified account name we wanted to use. But I thought `contract` is what made the most sense.

Reply:
In the unified scope we use:
- `spot`
- `margin`
- `future` (for settled futures contracts)
- `swap` (for perpetual futures aka perpetual swaps)
- `option` (for put option contracts and call option contracts)

Reply:
If you can, please, add the edits to match the above types.
Related: https://github.com/ccxt/ccxt/issues/11121